lib/src/base-styles/_nx-text-input.scss

```
@@ -117,11 +117,25 @@
 
 .nx-text-input--textarea {
   .nx-text-input__box {
-    align-items: flex-end;
+    align-items: unset;
+    display: flex;
+    flex-direction: column;
+
+    &::before {
+      content: unset;
+    }
+
+    svg {
```

We prefer to select by class
```suggestion
    .nx-icon {
```